orion.py

```diff
@@ -50,42 +55,45 @@ def orion(**kwargs):
     ES_URL=None
 
     if "ES_SERVER" in data.keys():
-        ES_URL = data['ES_SERVER']
+        ES_URL = data["ES_SERVER"]
     else:
-        if 'ES_SERVER' in os.environ:
-            ES_URL=os.environ.get("ES_SERVER")
+        if "ES_SERVER" in os.environ:
+            ES_URL = os.environ.get("ES_SERVER")
         else:
             logger.error("ES_SERVER environment variable/config variable not set")
             sys.exit(1)
-
+    shortener = pyshorteners.Shortener()
     for test in data["tests"]:
         uuid = kwargs["uuid"]
         baseline = kwargs["baseline"]
-        match = Matcher(index="perf_scale_ci", level=level, ES_URL=ES_URL)
+        match = Matcher(index="ospst-perf-scale-ci-*",
+                        level=level, ES_URL=ES_URL, verify_certs=False)
         if uuid == "":
             metadata = orion_funcs.get_metadata(test, logger)
         else:
             metadata = orion_funcs.filter_metadata(uuid,match,logger)
 
         logger.info("The test %s has started", test["name"])
         if baseline == "":
-            uuids = match.get_uuid_by_metadata(metadata)
+            runs = match.get_uuid_by_metadata(metadata)
+            uuids = [run["uuid"] for run in runs]
+            buildUrls = {run["uuid"]: run["buildUrl"] for run in runs}
             if len(uuids) == 0:
                 logging.info("No UUID present for given metadata")
                 sys.exit()
         else:
             uuids = [uuid for uuid in re.split(' |,',baseline) if uuid]
             uuids.append(uuid)
-        if metadata["benchmark.keyword"] == "k8s-netperf" :
+        if metadata["benchmark.keyword"] == "ospst-k8s-netperf" :
```

Comment:
I think we are going to need both ospst-k8s-netperf and k8s-netperf to work depending on which ES_SERVER we are looking at. Can we change the if statement to `if "k8s-netperf" in metadata["benchmark.keyword"]`

Reply:
@shashank-boyapally is work for allowing multiple ES_SERVERs part of a different PR?

Reply:
The discussion in the slack channel for allowing multiple ES_SERVERs in the same run is not yet developed, this just take cares for the internal instance changes we had.. will be developing that once I understand the further requirements more clearly.

Reply:
I still think the `if "k8s-netperf" in metadata["benchmark.keyword"]` would be better because this update breaks the usage of the QE elastic search index completely. This change would allow both to still work

Reply:
Yeah, I'll keep it as the previous, but the index what index would we like to search, as I believe in QE instance the index is `k8s-netperf` and in the internal instance it is `ospst-k8s-netperf`, should we include this index in the config file itself for better flexibility?

Reply:
yes I understand the index is different between QE instance and the internal instance. The code block I put in previous comments will help cover both instances. I think setting in the config will make it less flexible as we will have to have multiple files to pull the same type of data from each of the instances. That might need to be something addressed with the pr for setting multiple ES_SERVERS